### PR TITLE
Bop2coco converter

### PR DESCRIPTION
**Pull request by MartinSmeyer**
Adds a converter from BOP format to COCO format. Useful to interface with instance segmentation methods.

Related to #38

### CODE REVIEW

File: bop_toolkit_lib/pycoco_utils.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+    :param contour: The contour to close.
+    :return: The closed contour.
+    """
+    # If first != last point => add first point to end of contour to close it
+    if not np.array_equal(contour[0], contour[-1]):
+        contour = np.vstack((contour, contour[0]))
+    return contour
+
+def binary_mask_to_polygon(binary_mask, tolerance=0):
```
Comment (wangg12) [inline bop_toolkit_lib/pycoco_utils.py:91]: I think RLE is more suitable than polygons, since polygons can not deal with masks with holes.

Comment (MartinSmeyer) [reply bop_toolkit_lib/pycoco_utils.py:91]: I agree, we should probably add an option for RLE. 

Currently, the support for RLE is a bit questionable, I had problems using popular frameworks like Detectron2 with that format. See the discussions in
https://github.com/facebookresearch/detectron2/issues/200

But in future that hopefully changes.

Comment (wangg12) [reply bop_toolkit_lib/pycoco_utils.py:91]: It should be fixed now

**Review by thodan — APPROVED** [review]
Looking good to me, thanks!

**Pull request merged**